From 41d24ca505e43c7b0f49fbaade9fc8135d6ce206 Mon Sep 17 00:00:00 2001
From: "Mark A. Yoder" <Mark.A.Yoder@Rose-Hulman.edu>
Date: Wed, 26 Oct 2016 11:26:02 -0400
Subject: [PATCH] Unconfiged P9_12

---
 src/arm/univ-emmc-00A0.dts | 24 ++++++++++++++----------
 1 file changed, 14 insertions(+), 10 deletions(-)

diff --git a/src/arm/univ-emmc-00A0.dts b/src/arm/univ-emmc-00A0.dts
index ddb57ae..3951967 100644
--- a/src/arm/univ-emmc-00A0.dts
+++ b/src/arm/univ-emmc-00A0.dts
@@ -86,7 +86,7 @@
 /*      "P9.9",     PWR_BUT */
 /*      "P9.10",    RESETn  */
         "P9.11",
-        "P9.12",
+/*      "P9.12",    DS18B20 8?
         "P9.13",
         "P9.14",
         "P9.15",
@@ -747,14 +747,14 @@
                 pinctrl-single,pins = <0x070  0x36>; };     /* Mode 6, Pull-Up, RxActive */
 
             /* P9_12 (ZCZ ball U18) */
-            P9_12_default_pin: pinmux_P9_12_default_pin {
-                pinctrl-single,pins = <0x078  0x37>; };     /* Mode 7, Pull-Up, RxActive */
-            P9_12_gpio_pin: pinmux_P9_12_gpio_pin {
-                pinctrl-single,pins = <0x078  0x2F>; };     /* Mode 7, RxActive */
-            P9_12_gpio_pu_pin: pinmux_P9_12_gpio_pu_pin {
-                pinctrl-single,pins = <0x078  0x37>; };     /* Mode 7, Pull-Up, RxActive */
-            P9_12_gpio_pd_pin: pinmux_P9_12_gpio_pd_pin {
-                pinctrl-single,pins = <0x078  0x27>; };     /* Mode 7, Pull-Down, RxActive */
+//            P9_12_default_pin: pinmux_P9_12_default_pin {
+//                pinctrl-single,pins = <0x078  0x37>; };     /* Mode 7, Pull-Up, RxActive */
+//            P9_12_gpio_pin: pinmux_P9_12_gpio_pin {
+//                pinctrl-single,pins = <0x078  0x2F>; };     /* Mode 7, RxActive */
+//            P9_12_gpio_pu_pin: pinmux_P9_12_gpio_pu_pin {
+//                pinctrl-single,pins = <0x078  0x37>; };     /* Mode 7, Pull-Up, RxActive */
+//            P9_12_gpio_pd_pin: pinmux_P9_12_gpio_pd_pin {
+//                pinctrl-single,pins = <0x078  0x27>; };     /* Mode 7, Pull-Down, RxActive */
 
             /* P9_13 (ZCZ ball U17) */
             P9_13_default_pin: pinmux_P9_13_default_pin {
@@ -1534,7 +1534,8 @@
                 pinctrl-4 = <&P9_11_uart_pin>;
             };
 
-            P9_12_pinmux {
+/*
+P9_12_pinmux {
                 compatible = "bone-pinmux-helper";
                 status = "okay";
                 pinctrl-names = "default", "gpio", "gpio_pu", "gpio_pd";
@@ -1543,6 +1544,7 @@
                 pinctrl-2 = <&P9_12_gpio_pu_pin>;
                 pinctrl-3 = <&P9_12_gpio_pd_pin>;
             };
+*/
 
             P9_13_pinmux {
                 compatible = "bone-pinmux-helper";
@@ -2077,12 +2079,14 @@
                     dir-changeable;
                 };
 
+/*
                 P9_12 {
                     gpio-name = "P9_12";
                     gpio = <&gpio1 28 0>;
                     input;
                     dir-changeable;
                 };
+*/
 
                 P9_13 {
                     gpio-name = "P9_13";
-- 
2.1.4

