From 878d1ffc53a2722668c86c1f5b526f0d6cc895d8 Mon Sep 17 00:00:00 2001
From: "Mark A. Yoder" <Mark.A.Yoder@Rose-Hulman.edu>
Date: Wed, 2 Nov 2016 16:15:19 -0400
Subject: [PATCH] Removed P8_45 and P8_46

---
 src/arm/univ-emmc-00A0.dts | 146 ++++++++++++++++++++++-----------------------
 1 file changed, 73 insertions(+), 73 deletions(-)

diff --git a/src/arm/univ-emmc-00A0.dts b/src/arm/univ-emmc-00A0.dts
index 3951967..bbba310 100644
--- a/src/arm/univ-emmc-00A0.dts
+++ b/src/arm/univ-emmc-00A0.dts
@@ -72,8 +72,8 @@
         "P8.42", /*    hdmi    */
         "P8.43", /*    hdmi    */
         "P8.44", /*    hdmi    */
-        "P8.45", /*    hdmi    */
-        "P8.46", /*    hdmi    */
+//        "P8.45", /*    hdmi    */
+//        "P8.46", /*    hdmi    */
 
 /*      "P9.1",     GND     */
 /*      "P9.2",     GND     */
@@ -686,37 +686,37 @@
             P8_44_pwm_pin: pinmux_P8_44_pwm_pin {
                 pinctrl-single,pins = <0x0AC  0x23>; };     /* Mode 3, Pull-Down, RxActive */
 
-            /* P8_45 (ZCZ ball R1 ) hdmi    */
-           P8_45_default_pin: pinmux_P8_45_default_pin {
-                pinctrl-single,pins = <0x0A0  0x27>; };     /* Mode 7, Pull-Down, RxActive */
-            P8_45_gpio_pin: pinmux_P8_45_gpio_pin {
-                pinctrl-single,pins = <0x0A0  0x2F>; };     /* Mode 7, RxActive */
-            P8_45_gpio_pu_pin: pinmux_P8_45_gpio_pu_pin {
-                pinctrl-single,pins = <0x0A0  0x37>; };     /* Mode 7, Pull-Up, RxActive */
-            P8_45_gpio_pd_pin: pinmux_P8_45_gpio_pd_pin {
-                pinctrl-single,pins = <0x0A0  0x27>; };     /* Mode 7, Pull-Down, RxActive */
-            P8_45_pruout_pin: pinmux_P8_45_pruout_pin {
-                pinctrl-single,pins = <0x0A0  0x05>; };     /* Mode 5, Pull-Down*/
-            P8_45_pruin_pin: pinmux_P8_45_pruin_pin {
-                pinctrl-single,pins = <0x0A0  0x26>; };     /* Mode 6, Pull-Down, RxActive */
-            P8_45_pwm_pin: pinmux_P8_45_pwm_pin {
-                pinctrl-single,pins = <0x0A0  0x03>; };     /* Mode 3, Pull-Down*/
-
-            /* P8_46 (ZCZ ball R2 ) hdmi    */
-           P8_46_default_pin: pinmux_P8_46_default_pin {
-                pinctrl-single,pins = <0x0A4  0x27>; };     /* Mode 7, Pull-Down, RxActive */
-            P8_46_gpio_pin: pinmux_P8_46_gpio_pin {
-                pinctrl-single,pins = <0x0A4  0x2F>; };     /* Mode 7, RxActive */
-            P8_46_gpio_pu_pin: pinmux_P8_46_gpio_pu_pin {
-                pinctrl-single,pins = <0x0A4  0x37>; };     /* Mode 7, Pull-Up, RxActive */
-            P8_46_gpio_pd_pin: pinmux_P8_46_gpio_pd_pin {
-                pinctrl-single,pins = <0x0A4  0x27>; };     /* Mode 7, Pull-Down, RxActive */
-            P8_46_pruout_pin: pinmux_P8_46_pruout_pin {
-                pinctrl-single,pins = <0x0A4  0x05>; };     /* Mode 5, Pull-Down*/
-            P8_46_pruin_pin: pinmux_P8_46_pruin_pin {
-                pinctrl-single,pins = <0x0A4  0x26>; };     /* Mode 6, Pull-Down, RxActive */
-            P8_46_pwm_pin: pinmux_P8_46_pwm_pin {
-                pinctrl-single,pins = <0x0A4  0x03>; };     /* Mode 3, Pull-Down*/
+        //     /* P8_45 (ZCZ ball R1 ) hdmi    */
+        //   P8_45_default_pin: pinmux_P8_45_default_pin {
+        //         pinctrl-single,pins = <0x0A0  0x27>; };     /* Mode 7, Pull-Down, RxActive */
+        //     P8_45_gpio_pin: pinmux_P8_45_gpio_pin {
+        //         pinctrl-single,pins = <0x0A0  0x2F>; };     /* Mode 7, RxActive */
+        //     P8_45_gpio_pu_pin: pinmux_P8_45_gpio_pu_pin {
+        //         pinctrl-single,pins = <0x0A0  0x37>; };     /* Mode 7, Pull-Up, RxActive */
+        //     P8_45_gpio_pd_pin: pinmux_P8_45_gpio_pd_pin {
+        //         pinctrl-single,pins = <0x0A0  0x27>; };     /* Mode 7, Pull-Down, RxActive */
+        //     P8_45_pruout_pin: pinmux_P8_45_pruout_pin {
+        //         pinctrl-single,pins = <0x0A0  0x05>; };     /* Mode 5, Pull-Down*/
+        //     P8_45_pruin_pin: pinmux_P8_45_pruin_pin {
+        //         pinctrl-single,pins = <0x0A0  0x26>; };     /* Mode 6, Pull-Down, RxActive */
+        //     P8_45_pwm_pin: pinmux_P8_45_pwm_pin {
+        //         pinctrl-single,pins = <0x0A0  0x03>; };     /* Mode 3, Pull-Down*/
+
+        //     /* P8_46 (ZCZ ball R2 ) hdmi    */
+        //   P8_46_default_pin: pinmux_P8_46_default_pin {
+        //         pinctrl-single,pins = <0x0A4  0x27>; };     /* Mode 7, Pull-Down, RxActive */
+        //     P8_46_gpio_pin: pinmux_P8_46_gpio_pin {
+        //         pinctrl-single,pins = <0x0A4  0x2F>; };     /* Mode 7, RxActive */
+        //     P8_46_gpio_pu_pin: pinmux_P8_46_gpio_pu_pin {
+        //         pinctrl-single,pins = <0x0A4  0x37>; };     /* Mode 7, Pull-Up, RxActive */
+        //     P8_46_gpio_pd_pin: pinmux_P8_46_gpio_pd_pin {
+        //         pinctrl-single,pins = <0x0A4  0x27>; };     /* Mode 7, Pull-Down, RxActive */
+        //     P8_46_pruout_pin: pinmux_P8_46_pruout_pin {
+        //         pinctrl-single,pins = <0x0A4  0x05>; };     /* Mode 5, Pull-Down*/
+        //     P8_46_pruin_pin: pinmux_P8_46_pruin_pin {
+        //         pinctrl-single,pins = <0x0A4  0x26>; };     /* Mode 6, Pull-Down, RxActive */
+        //     P8_46_pwm_pin: pinmux_P8_46_pwm_pin {
+        //         pinctrl-single,pins = <0x0A4  0x03>; };     /* Mode 3, Pull-Down*/
 
 
             /************************/
@@ -1491,32 +1491,32 @@
                 pinctrl-5 = <&P8_44_pruin_pin>;
                 pinctrl-6 = <&P8_44_pwm_pin>;
             };
-            /* hdmi */
-            P8_45_pinmux {
-                compatible = "bone-pinmux-helper";
-                status = "okay";
-                pinctrl-names = "default", "gpio", "gpio_pu", "gpio_pd", "pruout", "pruin","pwm";
-                pinctrl-0 = <&P8_45_default_pin>;
-                pinctrl-1 = <&P8_45_gpio_pin>;
-                pinctrl-2 = <&P8_45_gpio_pu_pin>;
-                pinctrl-3 = <&P8_45_gpio_pd_pin>;
-                pinctrl-4 = <&P8_45_pruout_pin>;
-                pinctrl-5 = <&P8_45_pruin_pin>;
-                pinctrl-6 = <&P8_45_pwm_pin>;
-            };
-            /* hdmi */
-            P8_46_pinmux {
-                compatible = "bone-pinmux-helper";
-                status = "okay";
-                pinctrl-names = "default", "gpio", "gpio_pu", "gpio_pd", "pruout", "pruin","pwm";
-                pinctrl-0 = <&P8_46_default_pin>;
-                pinctrl-1 = <&P8_46_gpio_pin>;
-                pinctrl-2 = <&P8_46_gpio_pu_pin>;
-                pinctrl-3 = <&P8_46_gpio_pd_pin>;
-                pinctrl-4 = <&P8_46_pruout_pin>;
-                pinctrl-5 = <&P8_46_pruin_pin>;
-                pinctrl-6 = <&P8_46_pwm_pin>;
-            };
+            // /* hdmi */
+            // P8_45_pinmux {
+            //     compatible = "bone-pinmux-helper";
+            //     status = "okay";
+            //     pinctrl-names = "default", "gpio", "gpio_pu", "gpio_pd", "pruout", "pruin","pwm";
+            //     pinctrl-0 = <&P8_45_default_pin>;
+            //     pinctrl-1 = <&P8_45_gpio_pin>;
+            //     pinctrl-2 = <&P8_45_gpio_pu_pin>;
+            //     pinctrl-3 = <&P8_45_gpio_pd_pin>;
+            //     pinctrl-4 = <&P8_45_pruout_pin>;
+            //     pinctrl-5 = <&P8_45_pruin_pin>;
+            //     pinctrl-6 = <&P8_45_pwm_pin>;
+            // };
+            // /* hdmi */
+            // P8_46_pinmux {
+            //     compatible = "bone-pinmux-helper";
+            //     status = "okay";
+            //     pinctrl-names = "default", "gpio", "gpio_pu", "gpio_pd", "pruout", "pruin","pwm";
+            //     pinctrl-0 = <&P8_46_default_pin>;
+            //     pinctrl-1 = <&P8_46_gpio_pin>;
+            //     pinctrl-2 = <&P8_46_gpio_pu_pin>;
+            //     pinctrl-3 = <&P8_46_gpio_pd_pin>;
+            //     pinctrl-4 = <&P8_46_pruout_pin>;
+            //     pinctrl-5 = <&P8_46_pruin_pin>;
+            //     pinctrl-6 = <&P8_46_pwm_pin>;
+            // };
 
 
             /************************/
@@ -2057,20 +2057,20 @@ P9_12_pinmux {
                     input;
                     dir-changeable;
                 };
-                /* hdmi */
-                P8_45 {
-                    gpio-name = "P8_45";
-                    gpio = <&gpio2 6 0>;
-                    input;
-                    dir-changeable;
-                };
-                /* hdmi */
-                P8_46 {
-                    gpio-name = "P8_46";
-                    gpio = <&gpio2 7 0>;
-                    input;
-                    dir-changeable;
-                };
+                // /* hdmi */
+                // P8_45 {
+                //     gpio-name = "P8_45";
+                //     gpio = <&gpio2 6 0>;
+                //     input;
+                //     dir-changeable;
+                // };
+                // /* hdmi */
+                // P8_46 {
+                //     gpio-name = "P8_46";
+                //     gpio = <&gpio2 7 0>;
+                //     input;
+                //     dir-changeable;
+                // };
 
                 P9_11 {
                     gpio-name = "P9_11";
-- 
2.1.4

